include/ttmlir/Target/TTNN/program.fbs

```diff
@@ -99,6 +99,27 @@ table MatmulOp {
 }
 // ANCHOR_END: adding_an_op_matmul_fbs
 
+table Conv2dOp {
+  in0: tt.target.TensorRef;
+  in1: tt.target.TensorRef;
+  in2: tt.target.TensorRef;
```

Comment:
I would prefer it if we named these `input`, `weight`, `bias`, similiarly to embedding op above. It makes it easier to work with.